homeworks/andrew.markovskiy_MarkoAndrew93/Games/styles.css

```diff
@@ -0,0 +1,25 @@
+* {
+  box-sizing: border-box;
+  font-family: 'Roboto', sans-serif;
```

Comment:
use "" for font-family to pass lint:css

<img width="908" alt="Screenshot 2023-10-23 at 12 41 22" src="https://github.com/MastersAcademy/fe-course-basic-2023/assets/118203044/c2b79fe7-50bf-46e2-bff7-16bb93d6c5d8">